docs/social-media.md

```diff
@@ -1257,6 +1257,35 @@ YouTube 官方亦有提供频道 RSS，形如 <https://www.youtube.com/feeds/vid
 
 </Route>
 
+### 榜单与集合
+
+<Route author="5upernova-heng" example="/douban/list/subject_real_time_hotest" path="/douban/list/:type" :paramsDesc="['榜单类型，见下表。默认为实时热门书影音']">
+
+| 榜单/集合          | 路由（type）               |
+| ------------------ | -------------------------- |
+| 实时热门书影音     | subject_real_time_hotest   |
+| 影院热映           | movie_showing              |
+| 实时热门电影       | movie_real_time_hotest     |
+| 实时热门电视       | tv_real_time_hotest        |
+| 一周口碑电影榜     | movie_weekly_best          |
+| 华语口碑剧集榜     | tv_chinese_best_weekly     |
+| 全球口碑剧集榜     | tv_global_best_weekly      |
+| 国内口碑综艺榜     | show_chinese_best_weekly   |
+| 国外口碑综艺榜     | show_global_best_weekly    |
+| 虚构类小说热门榜   | book_fiction_hot_weekly    |
+| 非虚构类小说热门榜 | book_nonfiction_hot_weekly |
+| 热门单曲榜         | music_single               |
+| 华语新碟榜         | music_chinese              |
+| ...                | ...                        |
+
+> 上面的榜单/集合并没有列举完整。
+>
+> 如何找到榜单对应的路由参数：
+> 在豆瓣手机 APP 中，对应地榜单页面右上角，点击分享链接。链接路径 `subject_collection` 后的路径就是路由参数 `type`。
+> 如：小说热门榜的分享链接为：`https://m.douban.com/subject_collection/ECDIHUN4A`，其对应本 RSS 路由的 `type` 为 `ECDIHUN4A`，对应的订阅链接路由：`/douban/list/ECDIHUN4A`
```

Comment:
```suggestion
> 如：小说热门榜的分享链接为：`https://m.douban.com/subject_collection/ECDIHUN4A`，其对应本 RSS 路由的 `type` 为 `ECDIHUN4A`，对应的订阅链接路由：[`/douban/list/ECDIHUN4A`](https://rsshub.app/douban/list/ECDIHUN4A)
```

Reply:
It seems that if we conver this example to a link, then it can not be wrapped as code